### PR TITLE
Build from base images + add development Dockerfile

### PR DESCRIPTION
**Pull request by tanandy**
Related to https://github.com/OpenPaaS-Suite/openpaas-front/issues/2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,12 +102,15 @@ You can override standard build using docker build arguments
 Build frontends on the main branch and contact frontend on a custom branch `hantt12-patch-1`
 
 ```sh
-docker build --build-arg CONTACTS_GIT_BRANCH=hantt12-patch-1 -t openpaas-front .
+# Using BuildKit
+DOCKER_BUILDKIT=1 docker build --pull -f Dockerfile.dev --build-arg CONTACTS_GIT_BRANCH=hantt12-patch-1 -t openpaas-front-dev . --no-cache
```
Comment (alagane) [inline README.md]: Should change `CONTACTS_GIT_BRANCH` to `CONTACTS_GIT_TREEISH`

Comment (tanandy) [reply README.md]: good catch i change documentation

**Review by tanandy — COMMENTED** [review]
fixed

**Review by alagane — APPROVED** [review]

**Comment by tanandy** [comment]
cc @aduprat

**Review by MichaelBailly — APPROVED** [review]

**Pull request merged**